switchmate.py

```diff
@@ -6,16 +6,19 @@
 
 Usage:
 	./switchmate.py scan
+	./switchmate.py status
 	./switchmate.py <mac_address> auth
 	./switchmate.py <mac_address> <auth_key> switch [on | off]
 	./switchmate.py -h | --help
 """
 
-from __future__ import print_function
```

Comment:
The print function lets me remove the ending line return character more easily. I should probably have started with Python 3 anyway.